check-plugins/check_gitlab_token_expiration/check_gitlab_token_expiration.py

```diff
@@ -17,7 +17,9 @@
 
 parser.add_argument('--url', required=True)
 parser.add_argument('--private_token',required=True)
-parser.add_argument('--user_id',required=True)
+parser.add_argument('--user_id', default=None)
+parser.add_argument('--project_id', default=None)
+parser.add_argument('--group_id', default=None)
```

Comment:
WDYT using one argument `scope` and adding vlaues for each scope (https://docs.python.org/3/library/argparse.html#choices) then you would not need to do error handling.

Reply:
I changed the arguments from user_id, project_id and group_id to scope and id.